packages/snack-babel-standalone/.editorconfig

```diff
@@ -0,0 +1,11 @@
+# EditorConfig is awesome: http://EditorConfig.org
```

Comment:
is this different than eslint?

Reply:
editorconfig is basically a settings definition for your IDE. And it's supported by many, like vscode, jetbrains, vim, etc (usually through an additional plugin). So eslint lints code, editorconfig sets your IDE.